lang/tests/raw_dumps/statements.stderr

```diff
@@ -1,4 +1,7 @@
-[
-    found 't' at 100..101,
-]
+error: unexpected character
+  --> tests/raw_dumps/statements.ssl:11:1
+   |
+11 | type C1:C4 = $fruits
+   | ^ `t` was not expected here
```

Comment:
can chumsky produce more information like what could be legal here?

Reply:
oh definitely yeah, i just haven't hooked it up yet. the thing to look at is the error type - right now it's `Simple`, the thing you want is `Rich` https://docs.rs/chumsky/1.0.0-alpha.7/chumsky/error/struct.Rich.html